dcr/scenarios/ext-seq-multiple-dependencies/run.host.py

```diff
@@ -0,0 +1,27 @@
+from dcr.scenario_utils.test_orchestrator import TestFuncObj, TestOrchestrator
```

Comment:
This scenario has both(2 entry points). which runs first? or kicks off same time?